src/entity/prelude.rs

```diff
@@ -28,6 +28,50 @@ pub use chrono::NaiveDateTime as DateTime;
 #[cfg(feature = "with-chrono")]
 pub type DateTimeWithTimeZone = chrono::DateTime<chrono::FixedOffset>;
 
+/// Handles the time and dates in UTC
+///
+/// ### Example Usage
+/// ```ignore
+/// use chrono::{DateTime, NaiveDateTime, Utc};
+/// use sea_orm::prelude::*;
+///
+/// let my_model = fruit::Model {
+///        id: 3_i32,
+///        name: "Fruit".to_owned(),
+///        cake_id: Some(4),
+///        timer: DateTime::<Utc>::from_utc(NaiveDateTime::from_timestamp(61, 0), Utc),
+/// };
+///
+/// assert_eq!(
+///         fruit::Model {
+///             id: 3,
+///             name: "Fruit".to_owned(),
+///             cake_id: Some(4,),
+///             timer: DateTime::<Utc>::from_utc(NaiveDateTime::from_timestamp(61, 0), Utc),
+///         },
+///         my_model
+///     );
+///
+/// // Define a `Model` containing a type of `DateTimeUtc` field
+/// #[derive(Clone, Debug, PartialEq, DeriveModel, DeriveActiveModel)]
+/// pub struct Model {
+///     pub id: i32,
+///     pub name: String,
+///     pub cake_id: Option<i32>,
+///     pub timer: DateTimeUtc,
+/// }
+///
+/// #[derive(Copy, Clone, Debug, EnumIter, DeriveColumn)]
+/// pub enum Column {
+///     Id,
+///     Name,
+///     CakeId,
+///     Timer,
+/// }
+/// ```
+#[cfg(feature = "with-chrono")]
+pub type DateTimeUtc = chrono::DateTime<chrono::Utc>;
```

Comment:
The example would be inserting and updating `ActiveModel` with `DateTimeUtc` attribute.

e.g.

```rust
#[derive(Clone, Debug, PartialEq, DeriveEntityModel)]
#[sea_orm(table_name = "applog")]
pub struct Model {
    #[sea_orm(primary_key)]
    pub id: i32,
    pub action: String,
    pub json: Json,
    pub created_at: DateTimeWithTimeZone,
    pub deleted_at: DateTimeUtc,
}
```

Reply:
@billy1624 Some help on understanding the results of github action failures. They seem to build fine and store the values in the databases, but retrieval is an issue.

For example, asserting on mysql
```
2022-01-07T14:12:10.675164Z  INFO sqlx::query: /* SQLx ping */; rows: 0, elapsed: 5.909ms  
Error: Query("error returned from database: column \"launch_date\" is of type timestamp with time zone but expression is of type text")
thread 'main' panicked at 'assertion failed: `(left == right)`
  left: `1`,
 right: `0`: the test returned a termination value with a non-zero status code (1) which indicates a failure', /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/test/src/lib.rs:195:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

```

Reply:
The problem you encountered is rooted in https://github.com/SeaQL/sea-query/pull/222#discussion_r780303186